openedx/features/learner_profile/templates/learner_profile/learner_profile.html

```diff
@@ -25,18 +25,22 @@
 <main id="main" aria-label="Content" tabindex="-1">
     <div class="wrapper-profile">
         <div class="profile ${'profile-self' if own_profile else 'profile-other'}">
-            <div class="wrapper-profile-field-account-privacy"></div>
+            <div class="wrapper-profile-field-account-privacy">
+                % if own_profile and records_url:
+                    <div class="wrapper-profile-records">
+                        <a href="${records_url}">
+                            <button class="btn profile-records-button">${_("View My Records")}</button>
+                        </a>
+                    </div>
+                % endif
+            </div>
             % if own_profile:
                 <div class="profile-header">
                     <div class="header">${_("My Profile")}</div>
                     <div class="subheader">
                         ${_('Build out your profile to personalize your identity on {platform_name}.').format(
                             platform_name=platform_name,
                         )}
-                        % if records_url:
```

Comment:
Just double checking this text block if intended to be remove?

Reply:
Yeah. I'm moving the text paragraph with link up to the top right and converting it to a button. Button is Marco-approved, text was not.